pkg/util/csi/volume_snapshot_test.go

```diff
@@ -951,80 +952,121 @@ func TestGetVolumeSnapshotClass(t *testing.T) {
 		Driver: "bar.csi.k8s.io",
 	}
 
+	// storageclasses
+	fooStorageClass := &storagev1api.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+			Labels: map[string]string{
+				"velero.io/csi-volumesnapshot-class": "foo",
+			},
+		},
+		Provisioner: "foo.csi.k8s.io",
+	}
+
+	barStorageClass := &storagev1api.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "bar",
+			Labels: map[string]string{
+				"velero.io/csi-volumesnapshot-class": "foo",
+			},
+		},
+		Provisioner: "bar.csi.k8s.io",
+	}
+
+	hostpathStorageClass := &storagev1api.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "hostpath",
+			Labels: map[string]string{
+				"velero.io/csi-volumesnapshot-class": "foo",
+			},
+		},
+		Provisioner: "hostpath.csi.k8s.io",
+	}
+
+	blahStorageClass := &storagev1api.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "blah",
+			Labels: map[string]string{
+				"velero.io/csi-volumesnapshot-class": "foo",
+			},
+		},
+		Provisioner: "blah.csi.k8s.io",
+	}
+
 	objs := []runtime.Object{hostpathClass, fooClass, barClass, fooClassWithoutLabel, barClass2}
 	fakeClient := velerotest.NewFakeControllerRuntimeClient(t, objs...)
 
 	testCases := []struct {
-		name        string
-		driverName  string
```

Comment:
line 1075 of this file still references driverName.

Also shouldn't there be more test cases associated with this new functionality?